### ISSUE TITLE
View multiple logs at once in single window via docking

### ISSUE THREAD

**Issue by stephen-farris** [RolandPheasant/TailBlazer#173]
I would love to be able to watch multiple log files at once without in the same Tail Blazer instance without having to manually flip between tabs, similar to how an IDE lets you view multiple files and other views at once via docking.

**Comment by RolandPheasant** [comment]
You can drag the tabs out like on Google chrome

**Comment by stephen-farris** [comment]
Yeah, definitely!  Was just thinking it would be nice to be able to arrange them as docked panes within a single window.  Would be easier than manually sizing a number of individual windows.

**Comment by stephen-farris** [comment]
Or I guess another possibility that could solve the same need could be the ability to merge different log files into a single stream that you'd view like a single log file?

**Comment by stephen-farris** [comment]
(Assuming the log files are all related to one another anyway)
